apps/docs/src/guide/contracts/configurable-constants.md

```diff
@@ -0,0 +1,33 @@
+# Configurable Constants
+
+Sway introduces a powerful feature: configurable constants. When creating a contract, you have the possibility to define constants, each assigned with a default value.
+
+Prior to deploying the contract, you can then redefine the value for these constants, it can be all of them or as many as you need.
+
+This feature provides flexibility for dynamic contract environments. It allows a high level of customization, leading to more efficient and adaptable smart contracts.
+
+## Defining Configurable Constants
+
+Below is an example of a contract in which we declare four configurable constants:
+
+<<< @/../../docs-snippets/contracts/echo-configurables/src/main.sw#configurable-constants-1{rust:line-numbers}
+
+In this contract, we have a function `echo_configurables` that returns the values of the configurable constants.
+
+If each of theses constants have a new values have been assigned to them, the function will return the updated values. Otherwise, the function will return the default values.
```

Comment:
```suggestion
If each of these constants has new values that have been assigned to them, the function will return the updated values. Otherwise, the function will return the default values.
```